docs/index.html

```diff
@@ -262,7 +262,7 @@
 
 ```reason
 /* radius can be omitted */
-let drawCircle ::color ::radius=? () => {
+let drawCircle ::color radius::r=? () => {
```

Comment:
I think that in this case it's nicer to leave the `radius` argument as-is and change the `r` below into `radius` instead.